src/Components/Register/ManagerCustomerRegistration.js

```diff
@@ -0,0 +1,302 @@
+    {value: "NM", label: "NM"},
+    {value: "NY", label: "NY"},
+    {value: "NC", label: "NC"},
+    {value: "ND", label: "ND"},
+    {value: "OH", label: "OH"},
+    {value: "OK", label: "OK"},
+    {value: "OR", label: "OR"},
+    {value: "PA", label: "PA"},
+    {value: "RI", label: "RI"},
+    {value: "SC", label: "SC"},
+    {value: "SD", label: "SD"},
+    {value: "TN", label: "TN"},
+    {value: "TX", label: "TX"},
+    {value: "UT", label: "UT"},
+    {value: "VT", label: "VT"},
+    {value: "VA", label: "VA"},
+    {value: "WA", label: "WA"},
+    {value: "WV", label: "WV"},
+    {value: "WI", label: "WI"},
+    {value: "WY", label: "WY"},
+]
+
+export default class ManagerCustomerRegistration extends Component {
+
+    state = {
+        firstName: '',
+        lastName: '',
+        Username: '',
+        password: '',
+        password2: '',
+        address: '',
+        zipCode: '',
+        creditCard: '',
+        creditCardsList:[],
+        selectedCompany: {value: "None", label: "None"},
+        selectedState: null,
+    }
+    
+    getCompanies() {
+        //fetch list of companies
+        //dummy data
+        companies = [
+            {value: "None", label: "None"},
+            {value: "AMC", label: "AMC"},
+            {value: "Regal", label: "Regal"},
+            {value: "Carmike's", label: "Carmike's"}
+        ]
+        return companies;
+    }
+
+    setCompany = selectedCompany => {
+        this.setState({selectedCompany})
+    }
+    setSelectedState = selectedState => {
+        this.setState({selectedState})
+    }
+
+    handleChange = event => {
+        this.setState({[event.target.name]: event.target.value});
+    }
+    register() {
+        //Checking to make sure all of the fields are filled out correctly
+        console.log(this.state);
+        if(this.state.firstName === '' || 
+        this.state.lastname === '' || 
+        this.state.Username === '' || 
+        this.state.password === '' ||
+        this.state.address === '' ||
+        this.state.zipCode === '' ||
+        this.state.creditCard === '') {
+            window.alert("Please fill out all of the fields");
+        } else if(this.state.firstName.length > 128) {
+            window.alert("First name is too long");
+        } else if(this.state.lastName.length > 128) {
+            window.alert("Last name is too long");
+        } else if(this.state.Username.length > 128) {
+            window.alert("Username is too long");
+        } else if(this.state.password.length > 128) {
+            window.alert("Password is too long");
+        } else if(this.state.address.length > 128) {
+            window.alert("Address is too long");
+        } else if(this.state.zipCode.length !== 5) {
+            window.alert("Zip Code must be 5 characters long");
+        } else if(this.state.zipCode.match(/^[0-9]+$/) == null) {
+            window.alert("Zip Code must be only numbers");
+        } else if(this.state.creditCard.length < 10) {
```

Comment:
credit card is 16 digits

Reply:
Yeah I changed that in the registration branch

Reply:
Oh okay